forc/src/cli/commands/plugins.rs

```diff
@@ -1,8 +1,38 @@
+use crate::cli::PluginsCommand;
 use anyhow::Result;
+use clap::Parser;
+use std::ffi::OsStr;
+use std::path::PathBuf;
+
+/// Find all forc plugins available via `PATH`.
+///
+/// Prints information about each discovered plugin.
+#[derive(Debug, Parser)]
+pub struct Command {
+    /// Prints the absolute path to each discovered plugin.
+    #[clap(long = "verbose", short = 'v')]
+    print_full_path: bool,
+}
+
+pub(crate) fn exec(command: PluginsCommand) -> Result<()> {
+    let PluginsCommand { print_full_path } = command;
 
-pub(crate) fn exec() -> Result<()> {
     for path in crate::cli::plugin::find_all() {
-        println!("{}", path.display());
+        print_plugin(path, print_full_path);
     }
     Ok(())
 }
+
+fn print_plugin(path: PathBuf, print_full_path: bool) {
+    if print_full_path {
+        println!("{}", path.display());
+    } else {
+        println!(
+            "{}",
+            path.file_name()
+                .unwrap_or_else(|| OsStr::new(""))
```

Comment:
I think we should just `expect` here seeing as this is only used with paths yielded from looping over `plugin::find_all()` above which should never contain a file with no file name. I'd also add a comment to `print_plugin` function stating the assumption, and that it panics if this assumption isn't met.

Reply:
Makes sense! Was wondering how to best handle this to be honest